### PR TITLE
Issue #49 patch: Template selection dialog keeps shifting on selection -...

### PR DESCRIPTION
**Pull request by ravirnjn88**
Fixed the shifting of template selection dialogue on selection.
Screen shots: https://www.dropbox.com/s/7vrn6c7ap2x91et/BuildmLearn.jpg?dl=0

### CODE REVIEW

**Comment by martinrotter** [comment]
OK, I will check today or tomorrow and eventually merge.

**Comment by ravirnjn88** [comment]
@martinrotter Please review the PR

**Comment by martinrotter** [comment]
Will check today hopefully, I am totally busy.

2015-02-21 20:31 GMT+01:00 Ravi Ranjan <notifications@github.com>:

File: src/gui/formnewproject.ui [hunk 1]
```diff
@@ -24,10 +27,28 @@
...
+       <widget class="QListWidget" name="m_listTemplates"/>
+      </item>
+     </layout>
+    </widget>
+   </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
-      <string>Description of selected template</string>
+      <string>           Description of selected template        </string>
```
Comment (martinrotter) [inline src/gui/formnewproject.ui]: I really do not like those continuous spaces in this string. Fix this before I can even think about merging this pull request. It seems you try to hack centering the text of groupbox, which is not correct, each system does this on its own.

File: src/gui/formnewproject.ui [hunk 2]
```diff
@@ -6,13 +6,16 @@
...
-    <height>450</height>
+    <width>664</width>
+    <height>458</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Create new project</string>
   </property>
+  <property name="layoutDirection">
+   <enum>Qt::LeftToRight</enum>
```
Comment (martinrotter) [inline src/gui/formnewproject.ui]: Layout direction is always left to right, why you set it explicitly?

File: src/gui/formnewproject.ui [hunk 3]
```diff
@@ -24,10 +27,28 @@
      </property>
     </widget>
    </item>
+   <item row="0" column="0" rowspan="2">
+    <widget class="QGroupBox" name="groupBox_2">
+     <property name="title">
+      <string>          Available templates          </string>
```
Comment (martinrotter) [inline src/gui/formnewproject.ui]: Cannot accept those continuous spaces, sorry. :)

**Comment by ravirnjn88** [comment]
You are right, I am using those spaces for centering the text of the groupbox. While Using the alignment property (AlignCenter) it's not showing the result as expected.
Removed the spaces and corrected layout direction.

**Comment by martinrotter** [comment]
Looks better. Merging to master.

**Pull request merged**